.github/workflows/lint.yml

```
@@ -0,0 +1,63 @@
+name: Run linter on pull request
+on:
+  pull_request:
+    branches:
+      - '*'
+    paths-ignore:
+      - '.lib'
+      - '.husky'
+      - '.eslintrc'
+      - '.gitignore'
+      - '.travis.yml'
+      - 'Makefile'
+      - 'README.md'
+      - 'fields.json'
+      - 'mime.json'
+      - 'rid.json'
+      - 'rtype.json'
```

This action is supposed to run `lint`, so it would be easier to watch all Javascript files and only ignore the `.lib` directory

```suggestion
    paths:
      - '**.js'
      - '!.lib/**'
```